src/containers/WA_Groups/GroupConversations/GroupConversation.tsx

```diff
@@ -0,0 +1,215 @@
+import { ListItemButton } from '@mui/material';
+import { Link } from 'react-router-dom';
+import dayjs from 'dayjs';
+import { useApolloClient, useMutation } from '@apollo/client';
+
+import { COMPACT_MESSAGE_LENGTH, SHORT_DATE_FORMAT } from 'common/constants';
+import { MARK_AS_READ, CONTACT_FRAGMENT } from 'graphql/mutations/Chat';
+import { SEARCH_OFFSET } from 'graphql/queries/Search';
+import { WhatsAppToJsx } from 'common/RichEditor';
+import { MessageType } from 'containers/Chat/ChatConversations/MessageType/MessageType';
+import styles from './GroupConversation.module.css';
+import Track from 'services/TrackService';
+import { slicedString } from 'common/utils';
+import { AvatarDisplay } from 'components/UI/AvatarDisplay/AvatarDisplay';
+
+export interface GroupConversationProps {
+  contactId: number;
```

Comment:
```suggestion
  groupId: number;
```

Update prop names accordingly.